docs/index.md

```diff
@@ -69,32 +69,32 @@ Installing the latest jenkins plugin will create a config file (~/.steampipe/con
 connection "jenkins" {
   plugin = "jenkins"
 
-  # Get your API token from Jenkins https://www.jenkins.io/doc/book/using/using-credentials/
+  # The Jenkins server URL is required for all requests. Required.
+  # It should be fully qualified (e.g. # https://...) and point to the root of the Jenkins server location.
+  # Can also be set via the JENKINS_URL environment variable.
+  # server_url = "https://ci-cd.internal.my-company.com"
 
-  # url = "https://ci-cd.internal.my-company.com"
-  # user_id = "admin"
-  # api_token = "116af6f5cf749f31410983860c692850a2"
+  # The Jenkins username for authentication is required for requests. Required.
+  # Can also be set via the JENKINS_USERNAME environment variable.
+  # username = "admin"
+
+  # Either the password or the API token is required for requests. Required. 
+  # Can also be set via the JENKINS_PASSWORD environment variable.
+  # password = "aqt*abc8vcf9abc.ABC"
+
+  # Further information: https://www.jenkins.io/doc/book/using/using-credentials/   
```

Comment:
There isn't a official webpage showing how to get an API token, but this third-party website has a good guide: https://www.baeldung.com/ops/jenkins-api-token